docs/apispec-docs/src/main/scala/sttp/tapir/docs/apispec/schema/NameToSchemaReference.scala

```diff
@@ -5,6 +5,6 @@ import sttp.tapir.{Schema => TSchema}
 
 private[schema] class NameToSchemaReference(nameToKey: Map[TSchema.SName, ObjectKey]) {
   def map(name: TSchema.SName): Reference = {
-    Reference.to("#/components/schemas/", nameToKey(name))
+    nameToKey.get(name).fold(Reference.to(name.fullName, ""))(key => Reference.to("#/components/schemas/", key))
```

Comment:
It will be nice to have a comment here describing what's the case when we don't have a entry in the map for the given name, something like:
```
  def map(name: TSchema.SName): Reference = 
    nameToKey.get(name) match {
      // we have a reference to internal model  
      case Some(key) => Reference.to("#/components/schemas/", key)
      // we have a reference to external model  
      case None => Reference.to(name.fullName, "")
    }
```

Reply:
I added a comment.